doc/source/contributing/developer_guide.rst

```diff
@@ -88,7 +88,7 @@ using::
   :meth:`mpl:matplotlib.axes.Axes.plot`
 
 The `intersphinx_mappings`
-defined for ArviZ can be seen in `conf.py <https://github.com/arviz-devs/arviz/blob/main/doc/conf.py>`_.
+defined for ArviZ can be seen in `conf.py <https://github.com/arviz-devs/arviz/blob/main/doc/source/conf.py#L272>`_.
```

Comment:
```suggestion
defined for ArviZ can be seen in `conf.py <https://github.com/arviz-devs/arviz/blob/main/doc/source/conf.py>`_.
```

The line will probably change in a few weeks/months so I think it's better to not set it and make readers search for the `intersphinx_mappings` variable

Reply:
okay!